build.gradle

```diff
@@ -261,7 +261,10 @@ dependencies {
 
   // storage
   compile 'org.fusesource.leveldbjni:leveldbjni-all'
-  compile 'org.mapdb:mapdb'
+  compile('org.mapdb:mapdb') {
+    exclude group: 'net.jpountz.lz4', module: 'lz4'
```

Comment:
With dependency management this should be implicit, but it may help with snyk.io issues.